docs/user-guide/cli-using-benefits-of-team-config.md

```diff
@@ -0,0 +1,13 @@
+# Benefits of team configuration
+
+[Team configuration](../appendix/zowe-glossary#team-configuration) can make the initial setup of Zowe CLI more efficient by making service connection details easier to share and maintain within your organization.
+
+Consider the following benefits of using team configuration for roles across your dev team:
+
+- For a **team leader**, or Dev-Ops advocate, **sharing global team configurations** managed from one location, such as a software change management system. This allows multiple team members to use the same configurations stored in a repository or server.
+
+- For a Dev-Ops advocate or **team leader**, quickly **onboarding new application developers** by sharing the configuration file that your team uses with the new team member.
```

Comment:
quickly **onboarding new application developers**
to 
quickly **onboard new application developers**